ofl/redhatmono/OFL.txt

```diff
@@ -0,0 +1,93 @@
+Copyright 2018 The Fraunces Project Authors (https://github.com/undercasetype/Fraunces)
```

Comment:
This should be fixed

Reply:
I linked the OFL directly with packager, it is the actual file upstream. How could I miss it ?!

Reply:
I made a PR upstream